components/massmint/OverviewTable.vue

```diff
@@ -157,47 +157,24 @@ const props = withDefaults(
   defineProps<{
     disabled?: boolean
     nfts?: NFTS
+    collectionId?: string
   }>(),
   {
     disabled: false,
     nfts: undefined,
```

Comment:
Maybe make nfts default empty object 

Since you are doing


`props.nfts || {}`

A couple of times

Reply:
updated